1_create_streamflow_observations.ipynb

```diff
@@ -26,12 +26,15 @@
     "\n",
     "from nhm_helpers.nhm_hydrofabric import (\n",
     "    create_hru_gdf,\n",
+    "    create_segment_gdf,\n",
```

Comment:
This creates and error for me because this function doesn't take this many arguments